canvas_modules/harness/cypress/integration/properties/custom-controls.js

```diff
@@ -22,6 +22,11 @@ describe("Test of custom panels", function() {
 	});
 
 	it("Test custom panels are working correctly", function() {
+		// this next click is a workaround, the next click on the toggle won't trigger
+		// an onClick unless elsewhere on the panel is clicked first. remove when fixed
+		cy.get(".harness-custom-control-custom-toggle").first()
+			.find(".bx--form-item")
+			.click();
 		cy.get(".harness-custom-control-custom-toggle label").first()
```

Comment:
For code that possibly needs to future action can you add a TODO  string in the comment.

Reply:
added a TODO, thanks!